js/emoji_util.js

```diff
@@ -46,6 +120,7 @@
         if (!$el || !$el.length) {
             return;
         }
+
         $el.html(emoji.replace_unified($el.html()));
```

Comment:
Interesting - I'll spend a couple minutes digging into that history. To be perfectly honest, jQuery tricks like that make me feel dirty. Especially if we've already been working with the data in string form.